src/pkg/exporter/cmd/main.go

```diff
@@ -74,6 +74,13 @@ func forwardEvent(reportData *v1alpha1.ReportData) {
 			enabledOutputs = append(enabledOutputs, "openSearch")
 		}
 	}
+	if config.Governor.URL == "" || config.Governor.ClusterID == "" || config.Governor.CspSecretName == "" {
+		log.Error("Either URL or ClusterID or CSPSecretName is empty")
```

Comment:
Dont log error, instead we can simply go to else part. Suppose someone is working on openSearch, they would see this unwanted error.
Ex: 
  ``` 
  if config.Governor.URL != "" && config.Governor.ClusterID != "" && config.Governor.CspSecretName != "" {
    // Do governor stuff
  }